app/views/helpers/PenaltiesBanner.scala.html

```diff
@@ -55,10 +55,44 @@
   </a>
 }
 
-@if(penaltiesSummary.hasActivePenalties) {
-  @govukNotificationBanner(NotificationBanner(
-    title = Text(messages("penaltiesBanner.heading")),
-    content = HtmlContent(bannerContent),
-    titleId = Some("govuk-notification-banner-title-penalties-banner")
-  ))
+@noPenaltiesBannerContent = {
+<div class="govuk-notification-banner__heading" id="penalties-coming-banner-heading">
+    @messages("penaltiesComingBanner.heading")
+</div>
+<div class="govuk-body" id="penalties-coming-banner-content">
+    <p class="govuk-body" id="penalties-coming-first-para">@messages("penaltiesComingBanner.paraOne")</p>
+    <p class="govuk-body" id="penalties-coming-second-para">@messages("penaltiesComingBanner.paraTwo")</p>
+    <p class="govuk-body" id="penalties-coming-third-para">@messages("penaltiesComingBanner.paraThree")</p>
+    <a class="govuk-notification-banner__link" rel="noreferrer noopener" target="_blank" id="penalties-coming-link"
+       href="https://www.gov.uk/guidance/prepare-for-upcoming-changes-to-vat-penalties-and-vat-interest-charges">@messages("penaltiesComingBanner.linkText")</a>
+</div>
+}
+
+@bannerContent = @{
+    summary match {
```

Comment:
This works, but you have the same case match 3 times here. Could you simplify this by removing these bannerTitle and bannerContent blocks and putting the content in them straight in the case match at the bottom? Since you're already differentiating between the same 2 cases down there